rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java

```diff
@@ -3285,7 +3285,34 @@ void getBtcBlockchainInitialBlockHeight() throws IOException, VMException {
     }
 
     private BtcTransaction createTransaction() {
-        return new SimpleBtcTransaction(networkParameters, PegTestUtils.createHash());
+        return createTransaction(PegTestUtils.createHash());
+    }
+
+    private BtcTransaction createTransaction(Sha256Hash hash) {
+        return new SimpleBtcTransaction(networkParameters, hash);
+    }
+
+    private BtcTransaction createTransaction(int toPk, Coin value) {
+        return createTransaction(toPk, value, BtcECKey.fromPrivate(BigInteger.valueOf(123456)));
+    }
+
+    private BtcTransaction createUniqueTransaction(int toPk, Coin value) {
+        return createTransaction(toPk, value, new BtcECKey());
+    }
+
+    private BtcTransaction createTransaction(int toPk, Coin value, BtcECKey btcECKey) {
+        NetworkParameters params = NetworkParameters.fromID(NetworkParameters.ID_REGTEST);
+        BtcTransaction input = new BtcTransaction(params);
+
+        input.addOutput(Coin.FIFTY_COINS, btcECKey.toAddress(params));
```

Comment:
```suggestion
        input.addOutput(Coin.COIN, btcECKey.toAddress(params));
```
I Think We Should Reduce The Values To Reflect Our Daily Transactions Like Values

Reply:
Done